README.md

```diff
@@ -35,6 +35,25 @@ Add the service provider to your `config/app.php` file within the `providers` ke
 ],
 // ...
 ```
+### Configuration for Local Environment
+
+If you wish to only generators on your local environment, install via composer using --dev option.
+
+```shell
+composer require reliese/laravel --dev
+```
+
+Now add the provider in app/Providers/AppServiceProvider.php file.
+
+```shell
+public function register()
+{
+    if ($this->app->environment() == 'local') {
```

Comment:
Why don't we make this configurable? So we don't have to depend on another service provider.